scripts/swarm-fix-route_manager.sh

```diff
@@ -459,8 +476,8 @@ function main {
 
       d)
       echo "STEP 8:"
-      echo "Removing NAT UDP 514 exceptions in <<$INGRESS_NS>> namespace..."
-      delete_514_nat_exception $INGRESS_NS
+      echo "Removing NAT UDP/514 UDP/162 TCP/6514 exceptions in <<$INGRESS_NS>> namespace..."
```

Comment:
echo "Removing NAT UDP/514 UDP/162 TCP/514 TCP/6514 exceptions in <<$INGRESS_NS>> namespace..."